flow/scripts/variables.yaml

```diff
@@ -177,6 +177,22 @@ SYNTH_MEMORY_MAX_BITS:
   default: 4096
   stages:
     - synth
+SYNTH_MOCK_LARGE_MEMORIES:
+  description: >
+    Reduce memories larger than SYNTH_MEMORY_MAX_BITS to 1 row.
+
+    This is useful to separate the concern of instantiating and placing
+    memories from investigating other issues with a design.
+
+    Memories with a single 1 row will of course have unrealistically good
+    timing and area characteristics, but timing will still correctly terminate
+    in a register.
```

Comment:
Let me also point out a concern that some of the logic driving the address input on the RAM will be optimized out when we override the RAM size

Reply:
Possibly. In my experience people are surprised by what can get optimized out when they mock something in their design

Reply:
I see.... The idea is that whatever problems remain after mocking, they are real place/route/timing closure problems and worth sorting out while waiting for real RAM or some more accurate fake RAM.